server/reports/expiring-items/2_3_0/convert_data_js/package.json

```diff
@@ -11,7 +11,6 @@
   "types": "./convert_data.d.ts",
   "description": "",
   "devDependencies": {
-    "@extism/js-pdk": "^1.1.1",
```

Comment:
These types were not actually used, wanted to avoid confusion with extism-js version so removed